ApplicationLibCode/FileInterface/RifOpmCommonSummary.cpp

```diff
@@ -172,33 +188,59 @@ RiaDefines::EclipseUnitSystem RifOpmCommonEclipseSummary::unitSystem() const
 //--------------------------------------------------------------------------------------------------
 void RifOpmCommonEclipseSummary::buildMetaData()
 {
-    if ( m_eSmry )
+    std::vector<Opm::time_point> dates;
+    std::vector<std::string>     keywords;
+
+    if ( m_enhancedReader )
     {
-        auto dates = m_eSmry->dates();
-        for ( const auto& d : dates )
-        {
-            auto timeAsTimeT = std::chrono::system_clock::to_time_t( d );
-            m_timeSteps.push_back( timeAsTimeT );
-        }
+        dates    = m_enhancedReader->dates();
+        keywords = m_enhancedReader->keywordList();
+    }
 
-        auto [addresses, addressMap] = RifOpmCommonSummaryTools::buildMetaData( m_eSmry.get() );
+    if ( m_standardReader )
+    {
+        dates    = m_standardReader->dates();
```

Comment:
Dates and keywords from from m_standardReader will overwrite dates and keywords from m_enhancedReader. By design?